### PR TITLE
PostTools deprecations

### PR DESCRIPTION
**Pull request by mathematicalmichael**
this was a fresh checkout from master, merged @User-zwj's branch, and now everything looks nice and clean and proper, and all her commit history is still maintained. perfect.

### CODE REVIEW

File: test/test_postProcess/test_postTools.py [hunk 1]
```diff
@@ -128,9 +128,17 @@ def test_sort_by_rho(self):
         """
         Test :meth:`bet.postProcess.postTools.sort_by_rho`.
         """
-        (self.data, _) = postTools.sort_by_rho(self.data)
-        self.assertGreater(np.min(self.data.get_probabilities()), 0.0)
-        nptest.assert_almost_equal(np.sum(self.data.get_probabilities()), 1.0)
+        disc = sample.discretization(input_sample_set=self.data,
```
Comment (mathematicalmichael) [inline test/test_postProcess/test_postTools.py]: this extra test improves coverage ever so slightly. Just trying to get @User-zwj familiarized with the git workflow and this was a good starting place, testing support for discretizations passed as inputs.

Comment (mathematicalmichael) [reply test/test_postProcess/test_postTools.py]: perhaps add "and improve coverage" to title? in the future, will try to segment out changes into discrete PRs that only do one thing at a time.

**Comment by smattis** [comment]
I'm actually not sure why this deprecation warning is here. It seems to me like this code should still work, and could be useful. I think Lindley used it in a paper.

**Comment by smattis** [comment]
Looks good

**Pull request merged**